docs/team/user-roles.md

```diff
@@ -26,3 +26,38 @@ In some cases, the account `OWNER` will not be able to handle billing-related
 info and will want to invite one or more `BILLING_ADMIN`s who will be able to define, 
 edit, and delete credit cards and choose plans to be used for Apps in the 
 account.
+
+## Roles & Permissions
+
+### App Level Actions
+Action | App Owner | Admin | User
+--- | :---: | :---: | :---:
+Create [Alert Rule](https://sematext.com/docs/alerts/) | ✔️ | ✔️ | ✔️
+View [Alert Rules](https://sematext.com/docs/alerts/) created by Others | ✔️ | ✔️ | ✔️
+Edit/Delete/Disable [Alert Rule](https://sematext.com/docs/alerts/) created by them | ✔️ | ✔️ | ✔️
+Edit/Delete/Disable [Alert Rule](https://sematext.com/docs/alerts/) created by other users  | ✔️ | ✔️ | ✖️
+Ship Logs to an existing [Logs App](https://sematext.com/docs/logs/)| ✔️ | ✔️ | ✔️
+Ship Metrics to an existing [Monitoring/Infra App](https://sematext.com/docs/monitoring/) | ✔️ | ✔️ | ✔️
+Ship RUM data to an existing [Experience App](https://sematext.com/docs/experience/) | ✔️ | ✔️ | ✔️
+Create/Edit/Disable [Synthetics](https://sematext.com/docs/synthetics/getting-started/) Monitors | ✔️ | ✔️ | ✖️
+View [App](https://sematext.com/docs/guide/app-guide/) Usage Screen | ✔️ | ✔️ | ✖️
+Disable [Apps](https://sematext.com/docs/guide/app-guide/)  | ✔️ | ✔️ | ✖️
+Invite Others | ✔️ | ✔️ | ✖️
+Upgrade/downgrade Apps* | ✔️ | ✔️ | ✖️
```

Comment:
We can also add a footnote here instead of an `*`. So instead of
`Upgrade/downgrade Apps* `

we can write:
`Upgrade/downgrade Apps[^1]`

and it will look like:
Upgrade/downgrade Apps[^1]

[^1]: blah blah

Note that regardless where you put your note, it will always appear in the  end of the document (which is not bad imho).

Reply:
Aha great didn't know that, thanks Costas